api/app/signals/apps/email_integrations/rules/signal_handeld_negative_contact.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Gemeente Amsterdam
+from signals.apps.email_integrations.rules.abstract import AbstractRule
+from signals.apps.signals.models import Status
+from signals.apps.signals.workflow import AFGEHANDELD, VERZOEK_TOT_HEROPENEN
+
+
+class SignalHandledNegativeRule(AbstractRule):
```

Comment:
Please add a FEATURE_FLAG that allows us to enable/disable the sending of this email/
The feature flag can be added in the settings.FEATURE_FLAGS, for example:

`'REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED': os.getenv('REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED', True) in TRUE_VALUES,  # noqa`

In the rule just overwrite the `_validate` function. For example:

```
def _validate(self, *args, **kwargs):
    return settings.FEATURE_FLAGS.get('REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED', True) and super()._validate(*args, **kwargs)
```